spotify_player/src/event/page.rs

```diff
@@ -298,6 +298,25 @@ fn handle_command_for_context_page(
             ui.new_search_popup();
             Ok(true)
         }
+        Command::PlaySelectedPlaylist => {
+            if let PageState::Context {
+                context_page_type: ContextPageType::Browsing(context_id),
+                ..
+            } = ui.current_page()
+            {
+                if let ContextId::Playlist(_) = context_id {
+                    client_pub.send(ClientRequest::Player(PlayerRequest::StartPlayback(
+                        Playback::Context(context_id.clone(), None),
+                        None,
+                    )))?;
+                    Ok(true)
+                } else {
+                    Ok(false)
+                }
+            } else {
+                Ok(false)
+            }
+        }
```

Comment:
the logic doesn't seem right. You trigger this command in a playlist context page, so it's more like `PlayCurrentPlaylist`. I think this command's functionality is covered by `PlayRandom` command which supports all context not just playlist context.

For `PlaySelectedPlaylist` command, I expect that you can trigger it on a selected item in a playlist list. You do implement `Action::PlayContext` for it so probably don't need to define a separate command